### PR TITLE
Ingestion: FinNLP 2 volume

### PR DESCRIPTION
**Pull request by anthology-assist**
1. [x] Make sure the branch is merged with the latest `master` branch
1. [x] Ensure that there are editors listed in the `<meta>` block
1. [ ] If it's a workshop, add a `<venue>ws</venue>` tag
1. [x] Add events to their relevant SIGs
1. [x] Look at the venue listing for prior years, and ensure that the new volume titles are consistent. You can do this by clicking on the venue name from a paper page, which will take you to the vendor listing.
1. [x] Navigate to the event page preview (e.g., https://preview.aclanthology.org/icnlsp-ingestion/events/icnlsp-2021/), and page through, to see if there are any glaring mistakes
1. [x] Skim through the complete listing, looking for mis-parsed author names.
1. [x] Download the frontmatter and verify that the table of contents matches at least three randomly-selected papers
1. [x] Download 3–5 PDFs (including the first and last one) and make sure they are correct (title, authors, page numbers).

### CODE REVIEW

File: data/xml/2023.finnlp.xml [hunk 1]
```diff
@@ -204,4 +204,165 @@
       <bibkey>billert-conrad-2023-team</bibkey>
     </paper>
   </volume>
+  <volume id="2" ingest-date="2024-03-08" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of The 6th Workshop on Financial Technology and Natural Language Processing</booktitle>
```
Comment (mjpost) [inline data/xml/2023.finnlp.xml:209]: /
```suggestion
      <booktitle>Proceedings of the Sixth Workshop on Financial Technology and Natural Language Processing</booktitle>
```

Comment (anthology-assist) [reply data/xml/2023.finnlp.xml:209]: Done

**Comment by anthology-assist** [comment]
@mjpost ready for review

File: data/xml/2023.finnlp.xml [hunk 2]
```diff
@@ -204,4 +204,165 @@
...
+      <editor><first>Chung-Chi</first><last>Chen</last></editor>
+      <editor><first>Hen-Hsen</first><last>Huang</last></editor>
+      <editor><first>Hiroya</first><last>Takamura</last></editor>
+      <editor><first>Hsin-Hsi</first><last>Chen</last></editor>
+      <editor><first>Hiroki</first><last>Sakaji</last></editor>
+      <editor><first>Kiyoshi</first><last>Izumi</last></editor>
+      <publisher>Association for Computational Linguistics</publisher>
+      <address>Bali, Indonesia</address>
+      <month>November</month>
+      <year>2023</year>
```
Comment (mjpost) [inline data/xml/2023.finnlp.xml:219]: `<venue>ws</venue>

**Review by mjpost — APPROVED** [review]

**Pull request merged**